WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -538,12 +540,13 @@ public void onScrollCompleted() {
     }
 
     /*
-     * disable swipe-to-refresh if posts can be scrolled upwards - only necessary when the fragment
-     * toolbar is enabled since it can intercept the touch event and pass it on to the swipe layout,
-     * cause a refresh to begin even though the user is simply scrolling up the posts
+     * disable swipe-to-refresh if "new posts" bar is showing or posts can be scrolled up
+     * because otherwise it can intercept the touch event and pass it on to the swipe layout,
+     * causing a refresh to begin even though the user is simply scrolling up the posts.
+     * note that this problem only applies when the fragment toolbar is enabled.
      */
     private void checkSwipeToRefresh() {
-        mSwipeToRefreshLayout.setEnabled(!mRecyclerView.canScrollUp());
+        mSwipeToRefreshLayout.setEnabled(!mRecyclerView.canScrollUp() && !isNewPostsBarShowing());
```

Comment:
I was still able to do a PTR after switching to the Freshly Pressed tag and the new posts bar appeared:

https://cloudup.com/iOx611BxnOc